### PR TITLE
Companion for Adding benchmarking for new `frame_election_provider_support` #11149

### PR DESCRIPTION
**Pull request by georgesdib**
Companion for paritytech/substrate#11149

Added benchmarking to `frame_election_provider_support`.

### CODE REVIEW

File: runtime/kusama/src/weights/frame_election_provider_support.rs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	// Storage: Staking SlashingSpans (r:1 w:0)
+	// Storage: BagsList ListBags (r:200 w:0)
+	// Storage: BagsList ListNodes (r:2000 w:0)
+	// Storage: Staking Nominators (r:2000 w:0)
+	// Storage: Staking Bonded (r:2000 w:0)
+	// Storage: Staking Ledger (r:2000 w:0)
+	// Storage: Staking Validators (r:1001 w:0)
+	// Storage: System BlockWeight (r:1 w:1)
+	// Storage: Staking CounterForValidators (r:1 w:0)
+	// Storage: Staking ValidatorCount (r:1 w:0)
```
Comment (kianenigma) [inline runtime/kusama/src/weights/frame_election_provider_support.rs:41]: these values are wrong -- need to trigger the bench bot here after the PR.

**Review by kianenigma — APPROVED** [review]

**Review by bkchr — APPROVED** [review]
Runtime changes look reasonable. Trusting @kianenigma here ;)

**Review by dvdplm — APPROVED** [review]

**Comment by joao-paulo-parity** [comment]
@kianenigma please try the commands again, they should work after https://github.com/paritytech/bench-bot/pull/91

**Pull request merged**